Sources/CryptorECC/ECPrivateKey.swift

```diff
@@ -215,12 +292,140 @@ public class ECPrivateKey {
         }
         // If we stripped the leading zero earlier, add it back here
         var pubBytes = self.pubKeyBytes
-        if pubBytes.startIndex != 0 {
+        if stripped {
             pubBytes = Data(count: 1) + self.pubKeyBytes
         }
         return try ECPublicKey(der: keyHeader + pubBytes)
     }
+    
+    /// Decode this ECPrivateKey to it's PEM format
```

Comment:
Suggest making this private, and doing it during initialization so that the pem string is just a property.